File: problem_fifteen.js
// reverse text element
var value ="hello payer bhai"
function reverSe(val){
    var lst = val.split(" ")
    var arr =[]
    for(var i=0; i <lst.length;i++){
        arr.push(lst[i].split("").reverse().join(""))
    }
    var txt = " "
    for (var j =0; j<arr.length;j++){
        txt +=arr[j]

    }
    return txt
}
console.log(reverSe(value))

File: problem_fifteen.py
# reverse text element
val ="hello payer bhai"
def Reverse(value):
    lst = value.split(" ")
    arr =[]
    for i in lst:
        arr.append(i[::-1])   
    listToStr = ' '.join([str(elem) for elem in arr])          
    return  listToStr
print(Reverse(val))       

